src/openlcb/MemoryConfigClient.hxx

```diff
@@ -391,8 +391,17 @@ private:
         if (responseCode_ & DatagramClient::OPERATION_PENDING)
         {
             isWaitingForTimer_ = 1;
+            // Retrieve the timeout flags
+            uint8_t timeout_flags =
+                (dgClient_->result() >> DatagramClient::RESPONSE_FLAGS_SHIFT) &
+                    DatagramClient::REPLY_TIMEOUT_MASK;
+            // Convert the timeout flags to seconds. Timeout flags == 0 means
+            // no timeout, which we interpret as the OpenLCB intrinsic default
+            // of 3 seconds.
+            long long timeout_nsec = timeout_flags == 0 ?
+                 SEC_TO_NSEC(3) : SEC_TO_NSEC(0x1 << timeout_flags);
```

Comment:
should we use the existing constant defined [here](https://github.com/bakerstu/openmrn/blob/1064f5edbe3e526d3eb1c2fb8f596747f07d10ff/src/openlcb/Datagram.cxx#L42)?

Reply:
The update looks good and satisfies the consolidation of the usage of magic numbers for this use case to a single location. It would be good to similarly remove the referenced constant, likely in a similar manner.

Reply:
If you look at the solution, I did remove the "magic" number to a commented constexpr